packages/components/src/sandbox/index.js

```diff
@@ -104,14 +104,16 @@ class Sandbox extends Component {
 				}
 
 				observer = new MutationObserver( sendResize );
-				observer.observe( document.body, {
-					attributes: true,
-					attributeOldValue: false,
-					characterData: true,
-					characterDataOldValue: false,
-					childList: true,
-					subtree: true
-				} );
+				setTimeout( function() {
```

Comment:
This code fails when the observer triggers because `document.body` for some reason is `undefined`. Making it async solves the issue but it might have some unexpected side-effects. I'm worried that current JSDom implementation doesn't reflect the real scenario. There is also this possibility that we were mistakenly assuming that it works properly because test passes.